src/clients/vendor.rb

```diff
@@ -98,13 +100,17 @@ def main
       else
         Pkg.TargetInit("/", false)
 
-        @products = Y2Packager::Resolvable.find(kind: :product,
```

Comment:
This has not been a proper fix:
https://github.com/yast/yast-add-on/pull/88/files#diff-dcc0ccee4cc8bd2d2771647e7428b5cdL101
While testing the include I have found another crash which is fixed now.